plugins/extractors/service_yaml/README.md

```diff
@@ -0,0 +1,109 @@
+# service_yaml
+
+Extractor for Service from YAML file.
+
+## Usage
+
+```yaml
+source:
+  name: service_yaml
+  scope: services-stg
+  config:
+    file: "./path/to/service.meteor.yaml"
+    env_prefix: CI
+```
+
+## Inputs
+
+| Key          | Value    | Example               | Description                                                                                                        | Required? |
+|:-------------|:---------|:----------------------|:-------------------------------------------------------------------------------------------------------------------|:----------|
+| `file`       | `string` | `service.meteor.yaml` | File path of `service.yaml`                                                                                        | ✅         |
+| `env_prefix` | `string` | `CI`                  | Prefix for environment variables. These are made available as variables in `service.yaml` with the prefix trimmed. | ❌         |
+
+### `service.yaml` format
+
+```yaml
+name: "string" # REQUIRED
+id: "string" # REQUIRED
+team:
+  id: "string"
+  name: "string"
+  email: "string"
+description: "string"
+url: "string"
+version: "string"
+inputs: # OPTIONAL
+  - type: "batch" # ONE OF {batch, kafka}
+    platform: "bigquery" # If type is batch then bigquery 
+    options:
+      table_ref: "{project_id}:{dataset_id}.{table_id}" # REQUIRED
+  - type: "stream"
+    platform: "kafka" # If type is stream then kafka
+    options:
+      topic: "estimate-log" # REQUIRED
+      bootstrap_servers: "host:port" # REQUIRED
+outputs: # OPTIONAL
+  - type: "stream"
+    platform: "kafka"
+    options:
+      topic: "string"
+      bootstrap_servers: "host:port"
+create_time: "2006-01-02T15:04:05Z07:00"
+update_time: "2006-01-02T15:04:05Z07:00"
```

Comment:
These sample dates is unable to be parsed, we can remove the time offset

```suggestion
create_time: "2006-01-02T15:04:05Z"
update_time: "2006-01-02T15:04:05Z"
```

Reply:
Thanks for catching this!